spec/requests/bulk_export_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'spec_helper'
+
+describe BulkExportController do
+  let!(:admin) { create(:admin) }
+  let!(:regular_user) { create(:user) }
+  let!(:collection) { create(:collection, owner_user_id: admin.id) }
+  let!(:bulk_export) { create(:bulk_export, collection: collection, user: admin) }
+
+  describe '#index' do
+    let(:action_path) { bulk_export_index_path }
+    let(:subject) { get action_path }
+
+    it 'redirects when not logged in' do
+      subject
+      expect(response).to have_http_status(:redirect)
+      expect(response).to redirect_to(dashboard_path)
```

Comment:
Does a not logged in user have a dashboard_path?  (I'm not sure, but let's confirm.)